astroid/nodes/node_classes.py

```diff
@@ -612,6 +612,19 @@ class AssignName(
 
     _other_fields = ("name",)
 
+    assigned_stmts: Callable[
+        [
+            Union["AssignName", "AssignAttr"],
+            Union["List", "Tuple", "AssignName", "AssignAttr", None],
+            Optional[InferenceContext],
+            Optional[typing.List[int]],
+        ],
+        Any,
+    ]
+    """Returns the assigned statement (non inferred) according to the assignment type.
+    See astroid/protocols.py for actual implementation.
+    """
+
```

Comment:
What do you think about moving method types below `__init__` and `postinit`? I feel like where it's now, it could easily be mistaken for an attribute.